hasktorch/src/Torch/Typed/Functional.hs

```diff
@@ -620,16 +629,37 @@ symeig
      , shape'' ~ Square shape
      , SymeigDTypeIsValid device dtype
      )
-  => Bool -- ^ whether or not to calculate eigenvectors
-  -> Tri -- ^ upper or lower triagonal
+  => Tri -- ^ upper or lower triagonal
   -> Tensor device dtype shape -- ^ input
   -> ( Tensor device dtype shape'
      , Tensor device dtype shape''
      ) -- ^ eigenvalues and eigenvectors
-symeig eigenvectors upper input = unsafePerformIO
-  $ ATen.cast3 ATen.Managed.symeig_tbb input eigenvectors boolUpper
+symeig upper input = unsafePerformIO
+  $ ATen.cast3 ATen.Managed.symeig_tbb input True boolUpper
```

Comment:
To fix the issue of behaivor, I have deleted eigenvetors's flag and added symeigvalues.